bevy_lint/action.yml

```diff
@@ -9,6 +9,28 @@ description: |
 
   Note that this currently will override the default Rustup toolchain to nightly. You can fix this
   by calling `rustup default` after this action is run.
+inputs:
+  cache:
+    description: |
+      If set to `true`, this action will cache the built `bevy_lint` executable and reuse it on
+      subsequent runs, which can speed up the runtime. This defaults to `false`.
+    required: false
+    default: 'false'
```

Comment:
Note that this is a string and not a boolean. Action inputs are converted to strings, making these two steps equivalent:

```yml
- uses: TheBevyFlock/bevy_cli/bevy_lint@main
  with:
    cache: true

- uses: TheBevyFlock/bevy_cli/bevy_lint@main
  with:
    cache: 'true'
```

This is important when checking if the input is true or false, as non-empty strings are truthy:

```yml
if: ${{ true }} # true
if: ${{ 'true' }} # true
if: ${{ false }} # false
if: ${{ '' }} # false
if: ${{ 'false' }} # true! :O
```

This is why I do a lot of `${{ inputs.cache == 'true' }}` instead of just `${{ inputs.cache }}`.